Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -174,11 +169,22 @@ public class IonSerializationOptions
         public readonly bool PermissiveMode;
         
         public Dictionary<Type, IIonSerializer> IonSerializers { get; init; }
+        public Dictionary<string, object> CustomContext { get; init; }
+    }
+
+    public interface IIonSerializerFactory
+    {
+        IIonSerializer Create(IonSerializationOptions options, Dictionary<string, object> customContext);
     }
 
-    public interface IonSerializerFactory<T, TContext> where TContext : IonSerializationContext
+    public abstract class IonSerializerFactory<T> : IIonSerializerFactory
     {
-        public IonSerializer<T> create(IonSerializationOptions options, TContext context);
+        public abstract IonSerializer<T> Create(IonSerializationOptions options, Dictionary<string, object> customContext);
+
+        IIonSerializer IIonSerializerFactory.Create(IonSerializationOptions options, Dictionary<string, object> customContext)

```

Comment:
Was this also meant to be public?

Reply:
Please correct me if I'm wrong - we cannot use access modifiers when implementing interface explicitly and all explicitly implemented members can be accessed only through member of interface type, which is public,  so I don't think we need to make it public.

Reference: https://stackoverflow.com/questions/2669031/compilation-error-the-modifier-public-is-not-valid-for-this-item-while-expl

Reply:
TIL.